package.json

```diff
@@ -8,14 +8,13 @@
   "scripts": {
     "prepare": "husky",
     "preinstall": "npx only-allow pnpm",
+    "generate:models": "pnpm -r --if-present --stream run generate-models",
     "build": "turbo run build",
     "build:clean": "pnpm -r --if-present --stream run build:clean",
     "prettier": "prettier --write \"{packages,apps,__tests__,!dist}/**/*.{ts,js,json,md}\"",
     "prettier:check": "prettier --check \"{packages,apps,__tests__,!dist}/**/*.{ts,js,json,md}\"",
     "test:ci": "jest --config=jest.json",
-    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --verbose --config=jest.json --coverage=true --detectOpenHandles",
-    "db:migrate": "turbo run db:migrate",
-    "generate:models": "turbo run generate:models"
```

Comment:
This did not do it for me, only the pnpm version actually generated the sources

Reply:
That's probably because you are running the wrong script:

````json
{
  "generate:models": "pnpm -r --if-present --stream run generate-models",
}
```
Take a look at "pnpm -r --if-present --stream run generate-models". I should be "pnpm -r --if-present --stream run generate:models"